### ISSUE TITLE
Add option to choose to delete data in Research during Archiving process

### ISSUE THREAD

**Issue by DorienHuijser** [UtrechtUniversity/yoda-ansible#476]
#### Is your feature request related to a problem? Please describe.

From the data managers I hear that there are a lot of datasets which are both in the Vault as well as still in Research. Because most researchers archive data at the end of a project, it would make most sense if the active copy in Research would be deleted once the dataset has been fully archived (i.e. copied to the Vault). This would save a tremendous amount of storage space as well as costs and environmental footprint. 

#### Describe the solution you'd like

It would be nice if during the Archiving process, researchers can opt to have the active copy of the data package deleted in Research once the data package has been archived in the Vault. Once the data manager then accepts the data package for archiving and the full data package has been copied to the Vault, instead of unlocking the active version in Research, the package in Research will be deleted. 

If implementing this, the persons submitting a data package for Archiving should be informed that the active copy will be removed from Research, but that it's possible to copy data back to Research from the Vault. And that everyone in the research-group can still access the data in the Vault. The option to delete the active copy should be skippable as well, for those who do want to retain the data package.

#### Describe alternatives you've considered

The alternative is the current situation: not doing this and relying on researchers to clean up after themselves. But in practice this doesn't happen much and data managers have to contact researchers to ask if they need to retain the active copy. 

#### Additional context

This question came up during the updating of the Yoda website.

**Comment by DorienHuijser** [comment]
Edit: we can also steer a bit more and ask to choose between two options:
- Move to Vault (delete the Research version)

or

- Copy to Vault (retain version in Research)

**Comment by stsnel** [comment]
This is an interesting idea, thank you for the proposal. One of the data managers has already expressed support for this. We will request further input on this proposal at the next data manager meeting on 22 October.

**Comment by stsnel** [comment]
The proposal has been approved in yesterday's data manager meeting. Internal ticket number is now YDA-5988.

**Comment by erikvdbergh** [comment]
@stsnel Does this have a milestone or version target?

**Comment by stsnel** [comment]
@erikvdbergh 

Yes, thanks for the nudge. The current planning is to include this feature in Yoda 2.1.0. As with most features, priorities might change depending on input from stakeholders, and this can result in changes with respect to what features will be included in the release.

**Comment by lwesterhof** [comment]
Implemented in https://github.com/UtrechtUniversity/yoda-ruleset/commit/d6133e119351592b8d141e685ad51a6bbc841e0c

Will be include in v2.1.0.
